ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/ViewBasedReportJobDetails.tsx

```diff
@@ -13,31 +13,59 @@ import {
 
 import type { ViewBasedReportSnapshot } from 'services/ReportsService.types';
 import VulnerabilitySeverityIconText from 'Components/PatternFly/IconText/VulnerabilitySeverityIconText';
-import { getSearchFilterFromSearchString } from 'utils/searchUtils';
+import {
+    getSearchFilterFromSearchString,
+    getValueByCaseInsensitiveKey,
+    searchValueAsArray,
+} from 'utils/searchUtils';
+import { isVulnerabilitySeverity } from 'types/cve.proto';
+import { formatCveDiscoveredTime } from '../../utils/vulnerabilityUtils';
+
+const toTitleCase = (str: string) => {
+    return str.replace(/\w\S*/g, (txt) => txt.charAt(0).toUpperCase() + txt.slice(1).toLowerCase());
+};
```

Comment:
It isn't exactly the same, but would https://lodash.com/docs/#startCase work for this case just as well?

Reply:
Oh interesting. It looks like that third example wouldn't fully work for our case. We want to basically convert something like `Severity` and `SEVERITY` to `Severity`. I tried checking if there was a native JavaScript function for this. Surprised it doesn't exist.